### PR TITLE
Pass ATB param better

### PR DESCRIPTION
**Pull request by andrey-p**
**Reviewer:** @jdorweiler 
**CC:** @bsstoner 

## Description:

This gets the extension to pick up ATB param on install from the query string.

## Steps to test this PR:

On FF/Chrome:

1. Open a tab with a URL like: https://duckduckgo.com/?natb=v123-4ab
2. Install the extension
3. The ATB param will be `v123-4ab`.

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/background/chrome-events.es6.js [hunk 1]
```diff
@@ -9,9 +9,7 @@ const ATB = require('./atb.es6')
 chrome.runtime.onInstalled.addListener(function (details) {
     if (details.reason.match(/install/)) {
         ATB.updateATBValues()
-
-        // need to wait a bit for ATB to be set
-        setTimeout(() => ATB.openPostInstallPage(), 2000)
```
Comment (andrey-p) [inline shared/js/background/chrome-events.es6.js]: Nice that this timeout could be removed.

File: shared/js/background/chrome-wrapper.es6.js [hunk 2]
```diff
@@ -33,21 +33,19 @@ let mergeSavedSettings = (settings, results) => {
...
-            chrome.tabs.insertCSS(tab.id, {
-                file: '/public/css/noatb.css'
+            tabs.forEach(tab => {
+                chrome.tabs.insertCSS(tab.id, {
+                    file: '/public/css/noatb.css'
+                })
             })
-        }
+
+            resolve(tabs.map(tab => tab.url))
```
Comment (andrey-p) [inline shared/js/background/chrome-wrapper.es6.js]: I'm just passing the URLs from here, then reading them back from within the ATB module. That way it's easier to add tests.

File: shared/js/background/safari-wrapper.es6.js [hunk 3]
```diff
@@ -93,14 +93,9 @@ let mergeSavedSettings = (settings, results) => {
...
-    // and wait for the background process to tell us when to execute it
-    safari.application.activeBrowserWindow.tabs.forEach((tab) => {
-        if (tab.url.match(/https:\/\/([^/]+\.)?duckduckgo.com/)) {
-            tab.page.dispatchMessage('getATB')
-        }
-    })
+let getDDGTabUrls = () => {
+    // we don't currently support getting ATB from install page on Safari
+    return Promise.resolve([])
 }
```
Comment (andrey-p) [inline shared/js/background/safari-wrapper.es6.js]: It shouldn't be too difficult to actually return URLs from here, but I guess it's not a codepath we actually care about if Safari only does gallery installs now.

**Comment by jdorweiler** [comment]
👍

**Pull request merged**